src/main/rules/GCI108/python/GCI108.asciidoc

```diff
@@ -0,0 +1,62 @@
+= Prefer Efficient Classifiers: XGBoost vs RandomForest
+
+Using resource-heavy classifiers like `RandomForestClassifier` for standard classification tasks can result in longer execution times and higher carbon emissions.
+
+`XGBoost` offers a more optimized and eco-friendly alternative with competitive accuracy.
+
+== Metrics Comparison Table
+
+[cols="1,1,1,1", options="header"]
+|===
+|Classifier            |Accuracy |Time (s) |Carbon Emission (kg CO₂)
+
+|RandomForestClassifier
+|0.88
+|1.24
+|0.00091
+
+|XGBClassifier
+|0.89
+|0.47
+|0.00035
+|===
+
+XGBoost not only matches or exceeds the accuracy of RandomForest but also runs significantly faster and emits less CO₂.
+
+== Visual Comparison
```

Comment:
in your different graphics, you show several frameworks ... could you explain why do you choose XGBoost instead of RadomForestClassifier ... and not other framework ? only based on your array with metrics ?
for example, seeing your graphs ... HistGradientBoosting or ExtraTrees are not so bad, no ?